src/components/security_manager/include/security_manager/crypto_manager_impl.h

```diff
@@ -128,15 +133,13 @@ class CryptoManagerImpl : public CryptoManager {
   SSLContext* CreateSSLContext() OVERRIDE;
   void ReleaseSSLContext(SSLContext* context) OVERRIDE;
   std::string LastError() const OVERRIDE;
-  virtual bool IsCertificateUpdateRequired() const OVERRIDE;
+  virtual bool IsCertificateUpdateRequired(
```

Comment:
@AByzhynar remove virtual

Reply:
@LuxoftAKutsan Changed in eb15dd852a0ea418bfa444b74c8f15d792e254ad